docs/subset14.md

```diff
@@ -147,8 +147,9 @@ tuple-member-type-desc-list =
    | [ tuple-rest-desc ]
 tuple-rest-desc = type-desc "..."
 
-param-list = param ["," param]*
+param-list = param ["," param]* ["," rest-param]
 param = type-desc identifier
+rest-param = type-desc "..." [identifier]
```

Comment:
No. This was fine before. We don't have function type descriptors in the subset, so the identifier is required in the subset.

Reply:
fixed